index.html

```diff
@@ -0,0 +1,150 @@
+                                                    <button type="button"
+                                                        onclick="convertText(document.getElementById('textInput').value)"
+                                                        class="btn btn-outline-success btn-rounded z-depth-1a">Convert</button>
+                                                </div>
+                                            </div>
+
+
+                                        </div>
+
+                                        <label for="textOutput">Text Output:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="hexOutput" value=""
+                                            disabled placeholder="Text Output..."></textarea>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+    <script>
+
+        const hexInput = document.querySelector('#hexInput');
+        const textOutput = document.querySelector('#textOutput');
+        const errorMsg = document.querySelector('#errorMsg');
+        const hexInputValidation = new RegExp(/^([0-9A-Fa-f]{2})+$/g);
+
+        function convertHex(hexInput) {
+            errorMsg.innerHTML = "";
+            if(hexInput.match(hexInputValidation)){
+            let hex = hexInput.toString();
+            console.log(hex)
+            let str = '';
+            for (let i = 0; (i < hex.length && hex.substr(i, 2) !== '00'); i += 2) {
+
+                str += String.fromCharCode(parseInt(hex.substr(i, 2), 16));
+            }
+            textOutput.innerHTML = str;
+            }
+            else
+            {
+            errorMsg.innerHTML ="The entered value is not Hex value"
+            }
+        }
+
+        function convertText(textInput) {
+            console.log("function was triggered");
+            console.log(hexInput)
+            let arr = [];
+            for (let i = 0, l =textInput.length; i <l; i++){
+                let hex = Number(textInput.charCodeAt(i)).toString(16);
+                arr.push(hex)
+            }
+            hexOutput.innerHTML = arr.join('');
```

Comment:
~I personally am against modifying variables that are not defined within your scope. I'd move line 143 to~ (scratch that)

This line makes it look like you are accessing a variable that is called hexOutput and updating the innerHTML id use the DOM method to do this for easier radability